examples/imagenet/resnet_v1.py

```diff
@@ -51,10 +50,25 @@ def apply(self, x, filters, strides=(1, 1), train=True, dtype=jnp.float32):
 
 
 class ResNet(nn.Module):
-  """ResNetV1."""
+  """ResNet V1."""
+
+  # pylint: disable=arguments-differ
```

Comment:
Oh?

Reply:
aah my bad, I have a linter in VS code and it was complaining about it. I guess I will remove it.